core/types/header_ext_test.go

```diff
@@ -242,3 +242,55 @@ func TestHeaderSerializable_updates(t *testing.T) {
 		assert.Equal(t, wantedExtras, extras)
 	})
 }
+
+func TestHeaderExtraPostCopy(t *testing.T) {
+	t.Parallel()
+
+	t.Run("empty_extra", func(t *testing.T) {
+		t.Parallel()
+
+		got := &ethtypes.Header{
+			ParentHash: common.Hash{1},
+		}
+		extra := &HeaderExtra{}
+
+		extra.PostCopy(got)
+
+		want := &ethtypes.Header{
+			ParentHash: common.Hash{1},
+		}
+		wantExtra := &HeaderExtra{}
+		_ = WithHeaderExtra(want, wantExtra)
```

Comment:
What's the intention behind this line? I think it's a no-op.

Reply:
Nevermind. I wasn't expecting `assert.Equal()` to compare unexported values but now I see why it's necessary. It should, instead be refactored and I'll submit a suggestion in a moment.